src/index.js

```diff
@@ -1 +1,8 @@
 'use strict';
+
+const { createServer } = require('./createServer');
+
+createServer()
+  .listen(3000, () => {
+    global.console.log('Server is running on 3000 port');
```

Comment:
Instead of using 'global.console.log', it's more common to use 'console.log' directly.